package-res/resources/web/pentaho/visual/base/View.js

```diff
@@ -241,6 +458,24 @@ define([
       return null;
     },
 
+    /**
+     * Performs the visualization update loop.
+     *
+     * On each iteration,
+     * if the set of dirty property groups is empty, the update loop ends, successfully.
+     *
+     * Otherwise, the model is checked for validity, and if it is not valid, the update loop is rejected.
```

Comment:
Otherwise, the model is checked for validity and, if not valid, the update loop is rejected.